.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: false
 language: node_js
+addons:
+    code_climate:
+        repo_token: $CODECLIMATE_REPO_TOKEN
```

Comment:
Ok. Just FYI I rebase to the HEAD of master, so do a pull if you intend to work in this branch locally.